### PR TITLE
Remove ror_scl from tfm

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/foreman/tfm/tfm.spec [hunk 1]
```diff
@@ -199,7 +190,6 @@ export LD_LIBRARY_PATH=%{_libdir}\${LD_LIBRARY_PATH:+:\${LD_LIBRARY_PATH}}
 export MANPATH=%{_mandir}:\${MANPATH}
 export CPATH=%{_includedir}\${CPATH:+:\${CPATH}}
 export PKG_CONFIG_PATH=%{_libdir}/pkgconfig\${PKG_CONFIG_PATH:+:\${PKG_CONFIG_PATH}}
-export GEM_PATH=%{gem_dir}:\${GEM_PATH:+\${GEM_PATH}}\${GEM_PATH:-\`scl enable %{scl_ror} -- ruby -e "print Gem.path.join(':')"\`}
```
Comment (ehelms) [inline packages/foreman/tfm/tfm.spec]: We don't want to drop this as it sets the GEM_PATH for the tfm collection

File: packages/foreman/tfm/tfm.spec [hunk 2]
```diff
@@ -253,7 +243,6 @@ source scl_source enable %{scl_name}
 exec rake "\$@"
 EOF
 
-scl enable %{scl_ror} - << \EOF
```
Comment (ehelms) [inline packages/foreman/tfm/tfm.spec]: I think we want to switch this to `scl_ruby`

File: packages/foreman/tfm/tfm.spec [hunk 3]
```diff
@@ -21,7 +19,7 @@
 Summary: Package that installs %scl
 Name: %scl_name
 Version: 5.0
-Release: 11%{?dist}
+Release: 12%{?dist}
```
Comment (ehelms) [inline packages/foreman/tfm/tfm.spec]: I think dropping tfm-ror52 is enough to warrant a full version bump

Comment (zjhuntin) [reply packages/foreman/tfm/tfm.spec]: Thought that might be a thing, but good call!

**Pull request merged**